### ISSUE TITLE
Post Removal Reqeust - UW Homework Solutions

### ISSUE THREAD

**Issue by jsanders89** [adamalloy/Anagram#1]
Hello,

My name is Jonathan and I work with the introductory computer science courses at the University of Washington. This repository contains solution code to our Anagrams homework assignment. 

These posts' solution codes are indexed by Google, enabling current students to easily find them and cheat on their assignments.

Would you please remove your Anagram repository or make it private in order to help us enforce our academic policy?

By removing this repository, you would be removing a tool that our students could use in academic misconduct.

Thank you,
Jonathan Sanders
Intro Support, Computer Science & Engineering
University of Washington
jsanders@cs.washington.edu

More info on our course: http://courses.cs.washington.edu/courses/cse142/15su/

**Comment by adamalloy** [comment]
yes I can, I'll remove it ASAP. was trying to build up a repository for employers to look at some of my work, not encourage any type of misconduct.
